cadence/worker.py

```diff
@@ -15,7 +15,15 @@
 
 @dataclass
 class WorkerOptions:
-    pass
+    max_concurrent_activity_execution_size: int = None
+    worker_activities_per_second: float = None
+    max_concurrent_local_activity_execution_size: int = None
+    worker_local_activities_per_second: float = None
+    task_list_activities_per_second: float = None
+    max_concurrent_activity_task_pollers: int = None
+    max_concurrent_decision_task_execution_size: int = None
+    worker_decision_tasks_per_second: float = None
+    max_concurrent_decision_task_pollers: int = None
```

Comment:
The client-side worker options (such as `worker_activities_per_second` and `worker_decision_tasks_per_second`) aren't implemented yet in `cadence-python` I believe.

We may want to comment them out and note that for now, and allow contributors to enable them in the future.

And if in this PR (as we discussed outside of github) you're planning on only implementing the one option (`task_list_activities_per_second`), then that's probably the only one I would keep un-commented.